src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
Don't need to use fixed height for the card
```suggestion
```

Reply:
Still not fixed.